site/css/general.scss

```diff
@@ -60,6 +60,14 @@ i {
     font-style: italic;
 }
 
+sub {
+    @include subscript;
+}
+
+sup {
+    @include superscript;
+}
```

Comment:
As I have outlined in Slack, this sadly won't work well in the case that our fallback fonts are used (see the screenshot there).

I tried coming up with a CSS incantation that would work well in both scenarios, but sadly couldn't find one. Hope you have more luck! 🤞🏻

Reply:
Could this be made conditional with a @supports query and features-opentype or something like that?

Reply:
No, I don't think that'll work since `@supports` only indicates browser support, not whether the current font supports some feature.